imports/ui/components/input-row.test.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-env mocha */
+
+import React from 'react'
+import { shallow } from 'enzyme'
+import { expect } from 'meteor/practicalmeteor:chai'
+import InputRow from './input-row.jsx'
+import { Meteor } from 'meteor/meteor'
+
+if (Meteor.isClient) {
+  describe('InputRow', () => {
+    it('should render appropriately for the required props', () => {
+      const row = shallow(<InputRow label='test' identifier='test-ident' />)
+      // Checking the label fits
+      expect(row.find('label').text()).to.equal('test')
+      const inputEl = row.find('input')
+      // Checking the name and id attributes fit
+      expect(inputEl.props().name).to.equal('test-ident')
+      expect(inputEl.props().id).to.equal('test-ident')
+    })
+
+    it('should render additional classes on the input element, if specified', () => {
+      const row = shallow(<InputRow label='bla' identifier='bla1' inpAdditionalClass='class-test abc123' />)
```

Comment:
inpAdditionalClass is a class names string that's appended to the className value of the input that's contained in this component. The order doesn't matter, I'm just testing both classes are appended to the className value. Do you think I should make it more explicit of what it does?

Reply:
Ah, I see, makes sense. Was curious because the two regexes have different prefix/suffix patterns. To check if the individual class is included, I was wondering if both test could split on whitespace (`\s+`) and use `Array.prototype.includes` to check for each class.